config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -158,7 +158,7 @@ periodics:
           - --env=CL2_ENABLE_WATCH_LIST_FEATURE=true
           - --env=CONTAINER_IMAGE=registry-sandbox.k8s.io/pause:3.1 #TODO(ameukam): revert when registry.k8s.io is ready
```

Comment:
[off-topic]

hmm @ameukam, probably we can switch this job to use the private pull-through endpoint? most of the key scale jobs use this now and should be fine even if they are NAT-ed, at least from the image pull side of things.

... will file a tracking issue